### ISSUE TITLE
Third person camera should be able to rotate around to see the front of your avatar

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#3550]

**Comment by johnkslg** [comment]
Boosting in priority as its something streamers requested.

**Comment by johnkslg** [comment]
I messed around with this for a bit and heres what I found:
- There are two MouseLook components, one on the root player and one on HeadRotation, which separately handle rotations. We'll probably want to move the avatar out from under the root one (so create a new subobject from player that has the hierarchy, except the avatar which is a sibling).  Either that or change the target of mouse look to rotate something besides itself.
- When the player starts moving forward, youll want the avatar to lerp towards the direction the camera is facing. I left a stub in for this:
![image](https://user-images.githubusercontent.com/3536496/108929829-97432a80-75f9-11eb-8832-1d7af3ce054d.png)
Its in the UpdateRotation call, which is called by the kinematic system (third party tool) to allow us to modify rotation.  Ran into trouble getting this to work smart with the rotation of the player.  Youll need to get a good understanding of how all this works together and give it a clean solution, and clean it up while youre in there too.
